### ISSUE TITLE
context lines from buffer put in `q:` command-line window

### ISSUE THREAD

**Issue by 21eleven** [nvim-treesitter/nvim-treesitter-context#188]
##### version:
```
NVIM v0.8.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
```

##### `nvim-treesitter-context` commit:
[0dd5eae](https://github.com/nvim-treesitter/nvim-treesitter-context/commit/0dd5eae6dbf226107da2c2041ffbb695d9e267c1)

##### problem:
if you run `q:` to open the command-line window while `nvim-treesitter-context` is enabled then some buffer lines end up in your command window
![image](https://user-images.githubusercontent.com/8813855/202877653-0f93ffee-6799-455d-b698-f00f064d4c46.png)

**Comment by lewis6991** [comment]
Can you provide instruction on how to reproduce?

**Comment by soifou** [comment]
FWIW, I am able to reproduce this using nvim HEAD:

1 - `nvim -nu minimal.lua`
2 - once plugins installed, open a relatively long lua buffer
3 - scroll until nvim-treesitter-context hook some context
4 - open command-line window, either `q/` or `q:`

Notice the context has moved from the buffer to command-line window (see screenshot above).

Here is the `minimal.lua` using lazy.nvim:

```lua
local root = vim.fn.fnamemodify('./.repro', ':p')

-- set stdpaths to use .repro
for _, name in ipairs({ 'config', 'data', 'state', 'cache' }) do
    vim.env[('XDG_%s_HOME'):format(name:upper())] = root .. '/' .. name
end

-- bootstrap lazy
local lazypath = root .. '/plugins/lazy.nvim'
if not vim.loop.fs_stat(lazypath) then
    vim.fn.system({
        'git',
        'clone',
        '--filter=blob:none',
        'https://github.com/folke/lazy.nvim.git',
        '--branch=stable', -- latest stable release
        lazypath,
    })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
    {
        'nvim-treesitter/nvim-treesitter',
        build = ':TSUpdate',
        dependencies = {
            {
                'nvim-treesitter/nvim-treesitter-context',
                config = function()
                    require('treesitter-context').setup({})
                end,
            },
        },
        config = function()
            require('nvim-treesitter.configs').setup({
                ensure_installed = { 'lua' }
            })
        end,
    },
}
require('lazy').setup(plugins, {
    root = root .. '/plugins',
})

-- add anything else here
vim.opt.termguicolors = true
```

**Comment by lewis6991** [comment]
Minimal minimal.lua with lazi.nvim:

```lua
vim.opt.runtimepath:append('.../nvim-treesitter')
vim.opt.runtimepath:append('.../nvim-treesitter-context')

require('treesitter-context').setup()
```

Looks like this was closed in #200
